README.md

```diff
@@ -1,132 +1,88 @@
-# lock-screen explainer
+# Lock Screen API Explainer
 
-This is the repository for lock-screen. You're welcome to
-[contribute](CONTRIBUTING.md)!
+Authors: Louise Brett, Glen Robertson
 
-To fill this out, please see: https://github.com/w3ctag/w3ctag.github.io/blob/master/explainers.md
-## Authors:
+## Overview
 
-- [Author 1]
-- [Author 2]
-- [etc.]
+This document proposes a Lock Screen API for enabling web apps on the lock screen of a device. This would allow users to use web apps for tasks such as checking a calendar, using a calculator, taking a photo, or taking a note, all without going through the extra step to unlock their device first. Such an API would be applicable to both desktop and mobile platforms, though it will require some operating system support to implement it.
 
-## Participate
-- [Issue tracker]
-- [Discussion forum]
+When apps are run on the lock screen of a device, access to user data should be thoughtfully considered and restricted. For example a camera app running on the lock screen may let the user take photos but not view existing ones, and a calendar app on the lock screen may let the user see an upcoming event but not others. So the proposal includes isolation of lock screen apps, with methods to pass data to and from the lock screen to reduce the risk of apps accidentally exposing user data.
 
-## Table of Contents [if the explainer is longer than one printed page]
+## Proposed API
 
-[You can generate a Table of Contents for markdown documents using a tool like [doctoc](https://github.com/thlorenz/doctoc).]
+Apps that want to be shown on the lock screen would indicate this by declaring a URL in their manifest, to be used to launch the app on the lock screen. The URL must be within the _scope_ defined in the manifest. This URL could be a new manifest entry such as _lock\_screen\_start\_url_ but a more generic configuration would allow other custom launch actions to be added in future. We propose adding a _special\_launchers_ list to the manifest, in which items declare a _URL_ and a _purpose_ that may be recognised by user agents.
```

Comment:
This is now outdated. Consider:
```suggestion
Apps that want to be shown on the lock screen would indicate this by declaring a URL in their manifest, to be used to launch the app on the lock screen. The URL must be [within the scope](https://www.w3.org/TR/appmanifest/#dfn-within-scope) defined in the manifest. This URL could be a new manifest entry such as `lock_screen_start_url` or a more-extensible `lock_screen` dict with a `start_url` member (proposed).
```
(also replace italics with codeblocks for code)